spec/features/course/assessment/submissions_viewing_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe 'Course: Submissions Viewing' do
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+    let(:assessment) { create(:course_assessment_assessment, course: course) }
+    before { login_as(user, scope: :user) }
+
+    context 'As a Course Manager' do
+      let(:user) { create(:course_manager, :approved, course: course).user }
+      let(:students) { create_list(:course_student, 3, :approved, course: course) }
+      let(:attempting_submission) do
```

Comment:
I think convention states that these local variables be constructed explicitly in the scenario instead of in let blocks.